pandas/tests/plotting/test_backend.py

```diff
@@ -46,14 +46,18 @@ def test_backend_is_correct(monkeypatch):
 
 @td.skip_if_no_mpl
```

Comment:
why doesn't this work in this case?

Reply:
The issue is running the tests from a non-virtualenv-like setup.  So matplotlib _is_ installed, but the pandas being tested is not in site-packages